generated/nidaqmx/constants.py

```diff
@@ -269,9 +273,9 @@ class Edge(Enum):
 
 
 class EncoderType(Enum):
-    X_1 = 10090  #: If signal A leads signal B, count the rising edges of signal A. If signal B leads signal A, count the falling edges of signal A.
-    X_2 = 10091  #: Count the rising and falling edges of signal A.
-    X_4 = 10092  #: Count the rising and falling edges of signal A and signal B.
+    X1 = 10090  #: If signal A leads signal B, count the rising edges of signal A. If signal B leads signal A, count the falling edges of signal A.
+    X2 = 10091  #: Count the rising and falling edges of signal A.
+    X4 = 10092  #: Count the rising and falling edges of signal A and signal B.
```

Comment:
This is fine

Reply:
`EncoderType.X_4` is used in nidaqm-python repo. Thus have retained the expected enum value name (`X_4`)